pkg/network/kubevirt_ipam_controller.go

```diff
@@ -49,6 +49,7 @@ func renderKubevirtIPAMController(conf *cnao.NetworkAddonsConfigSpec, manifestDi
 	}
 	data.Data["IsOpenshift"] = clusterInfo.OpenShift4
 	data.Data["EnableSCC"] = clusterInfo.SCCAvailable
+	data.Data["EnableNetworkAttachmentDefinition"] = clusterInfo.NetAttachDefAvailable
```

Comment:
if the CRD is NA, there will be a fatal https://github.com/kubevirt/cluster-network-addons-operator/pull/1841/files#diff-463a5613afbb95588d747500e304c2d348c7fce8ec35117067322e7aac36a8dcR106
so basically this isn't needed (just need to fatal if CRD is NA + conf.ipam == true)
the only advantage of it, is that we can disable it easily if needed
wdyt ?

Reply:
I think it was discussed that we want to fail early.. totally forgot. will fix. thanks!